policykit/policykit/settings.py

```diff
@@ -37,6 +37,7 @@
                  'ec2-54-190-197-117.us-west-2.compute.amazonaws.com',
                  '54.190.197.117']
 
+METAGOV_ENABLED = True if METAGOV_URL else False
```

Comment:
@rickygv99 @amyxzhang also adding this `METAGOV_ENABLED` flag in settings, and making sure we check it anywhere we do Metagov things in policy engine. This ensures that PolicyKit still works without Metagov too. (I had introduced a bug that @rickygv99 caught with his recent PR, thansk! this should make it easier going forward)